packages/nouns-webapp/src/wagmi.ts

```diff
@@ -13,15 +13,20 @@ const activeChain =
     find(chain => chain.id === activeChainId),
   ) ?? sepolia;
 
-const mainnetRpcUrl = import.meta.env.VITE_MAINNET_JSONRPC ?? '';
-const sepoliaRpcUrl = import.meta.env.VITE_SEPOLIA_JSONRPC ?? '';
+const transports = {
+  [mainnet.id]: fallback([
+    webSocket(import.meta.env.VITE_MAINNET_WSRPC ?? ''),
```

Comment:
if VITE_MAINNET_WSRPC is unset, it breaks instead of falling back:
```
chunk-E3PFJVWP.js?v=383e6a62:23596 Uncaught UrlRequiredError: No URL was provided to the Transport. Please provide a valid RPC URL to the Transport.
```
you have to check if the env is set and conditionally add the transport instead